src/main/java/banking/CardValidator.java

```diff
@@ -0,0 +1,11 @@
+package banking;
+
+public class CardValidator {
+
+    public boolean isValidCard(CardGenerator generator, String card) {
+        if (card.length() < 16) return false;
```

Comment:
Oracle Java code convention recommends always using curly brackets, please follow it.
```
if (condition) {
 doSomething();
}
```

Reply:
fixed

Reply:
I still see `if (card.length() < 16) return false;`.

Reply:
fixed